src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -165,12 +165,12 @@ public List<Path> getFileDirectories(FilePreferences preferences) {
         Optional<Path> userFileDirectory = metaData.getUserFileDirectory(preferences.getUserAndHost()).map(dir -> getFileDirectoryPath(dir));
         userFileDirectory.ifPresent(fileDirs::add);
 
-        Optional<Path> generalFileDirectory = metaData.getDefaultFileDirectory().map(dir -> getFileDirectoryPath(dir));
+        Optional<Path> generalFileDirectory = metaData.getLibraryFileDirectory().map(dir -> getFileDirectoryPath(dir));
         generalFileDirectory.ifPresent(fileDirs::add);
 
         // fileDirs.isEmpty() is true after these two if there are no directories set in the BIB file itself:
         //   1) no user-specific file directory set (in the metadata of the bib file) and
-        //   2) no general file directory is set (in the metadata of the bib file)
+        //   2) no library file directory is set (in the metadata of the bib file)
```

Comment:
When reading this,  I think. everywhere, the “-specific“ should be added. Also in constants, getters, ...